### PR TITLE
KFSPTS-19396 add fix to use isNull criteria instead of equal to empty…

### PR DESCRIPTION
**Pull request by db483**
… String as this does not work in Oracle DB

### CODE REVIEW

File: src/main/java/org/kuali/kfs/fp/document/dataaccess/impl/GeneralLedgerTransferEntryDaoOjb.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        }
+
+        return criteria;
+    }
+
+    public boolean hasTransferredEntries(String documentNumber) {
+        Criteria criteria = new Criteria();
+        criteria.addEqualTo(KFSPropertyConstants.DOCUMENT_NUMBER, documentNumber);
+        criteria.addEqualTo(KFSPropertyConstants.FINANCIAL_SYSTEM_ORIGINATION_CODE, KFSConstants.ORIGIN_CODE_KUALI);
+        criteria.addNotNull(KFSPropertyConstants.GENERAL_LEDGER_TRANSFER_DOCUMENT_NUMBER);
```
Comment (jhulslander) [inline src/main/java/org/kuali/kfs/fp/document/dataaccess/impl/GeneralLedgerTransferEntryDaoOjb.java:159]: is this also a CU customization?  If so, should we add a note here as well?

**Review by jhulslander — COMMENTED** [review]
code changes look good, I had one question though

File: src/main/java/org/kuali/kfs/fp/document/dataaccess/impl/GeneralLedgerTransferEntryDaoOjb.java (hunk 1 above)
Comment (db483) [reply src/main/java/org/kuali/kfs/fp/document/dataaccess/impl/GeneralLedgerTransferEntryDaoOjb.java:159]: This is not CU customization, the only change to this file is the line I have marked.

**Review by jhulslander — APPROVED** [review]
Thanks for confirming, this unit tests pass, I shall merge

**Pull request merged**